deploy/non-olm/v1.0

```diff
@@ -0,0 +1 @@
+v1.0.1
```

Comment:
This link makes it so that once we update the Dockerfile downstream for v1.0.1 we won't have to remember to do this for every release. It also means we don't have to find all the doc locations where v1.0.x was referenced and update them. I think I caught them all, but in the invent I didn't we can catch them and fix them once.